src/monodroid/jni/build-info.hh

```diff
@@ -17,13 +17,6 @@ namespace xamarin::android::internal
 		static constexpr char xa_version[] = XA_VERSION;
 		static constexpr char date[] = __DATE__;
 
-		static constexpr bool is_net6 =
```

Comment:
@grendello I couldn't find references to this anywhere, should this be restored and renamed or is it not needed?

Reply:
It can be removed. I think I used it at one point in code that's gone now, so no need to keep it around.